### DISCUSSION TITLE
Mario

### DISCUSSION THREAD

**Discussion by PlayGame666** [skyline-emu/skyline#2053] [Game Requests]
I would love to play new super mario bros U DELUXE this is my childhood game and I really want to play my phone is able to run this game I checked it on a bad emulator egg ns it was 25-50 fps but I know that it is bad and not optimized for this I think  that on this emulator the game on my phone will fly, and so I will say a big thank you to the developers of this emulator, because this emulator began to run my favorite games.

I wrote through a translator on this, sorry for the mistakes

**Comment by Mononono87** [comment]
It's a 32 bit game and won't run for a long while.also don't use egg ns to compare games

**Reply by PlayGame666** [reply]
Okay
